radio/src/telemetry/telemetry.cpp

```diff
@@ -381,7 +374,6 @@ void telemetryReset()
   telemetryData.hub.current = 55;
   telemetryData.hub.maxCurrent = 65;
 #endif
-#endif
 
 /*Add some default sensor values to the simulator*/
 #if defined(CPUARM) && defined(SIMU)
```

Comment:
Indeed, that looked more recent so thought there may be a reason - but yeah no point.